sensor/common/image/service_impl.go

```diff
@@ -85,6 +109,32 @@ func (s *serviceImpl) GetImage(ctx context.Context, req *sensor.GetImageRequest)
 	}, nil
 }
 
+// createNoAuthDockerRegistry creates a new registry image integration of type docker that
+// has no credentials
+//
+// TODO: temporary until image integrations from central are used instead
+func createNoAuthDockerRegistry(ctx context.Context, registry string) (registryTypes.Registry, error) {
```

Comment:
what is the difference between this and just using `UpsertNoAuthRegistry`?

Reply:
`UpsertNoAuthRegistry` requires a namespace to upsert into the registry store - this code is executed by the admission controller workflow that currently does not provide namespace (or any deployment) details. 

If preferred I can move the creation of the no auth registry to a utility method and use it in both places, my thinking was since this is temporary not to over complicate it. Thoughts?

Reply:
`UpsertNoAuthRegistry` has been removed in latest commits